web/pdf_attachment_viewer.js

```diff
@@ -116,13 +116,12 @@ class PDFAttachmentViewer {
   /**
    * @param {PDFAttachmentViewerRenderParameters} params
    */
-  render(params = {}) {
-    var attachments = params.attachments || null;
-    var attachmentsCount = 0;
+  render({ attachments, keepRenderedCapability = false, }) {
```

Comment:
There's a comment in https://github.com/mozilla/pdf.js/blob/9e4dcf156b6d31926b8d48112ee6625b674b1ddc/web/pdf_attachment_viewer.js#L55-L59, which was added as a result of the discussion in https://github.com/mozilla/pdf.js/pull/7959#discussion_r96517486.

Since that parameter should *not* be used unless you know what you're doing, I didn't want to add a too visible comment; do you no longer think that this was the right call here?